### PR TITLE
Task/login modal

### PR DESCRIPTION
**Pull request by garr741**
Probably do that older PR for Professional Experience View before this because this one includes it.

### CODE REVIEW

**Comment by hkiang01** [comment]
That's a lot of commits

File: imports/ui/components/Navbar.jsx [hunk 1]
```diff
@@ -50,7 +50,11 @@ class Navbar extends Component {
   }
 
   handlePeopleClicked() {
-    browserHistory.push('/users')
+    browserHistory.push('/users/1')
```
Comment (bwencke) [inline imports/ui/components/Navbar.jsx]: this should be /users

File: imports/ui/views/login/LoginModal/CreateAnAccountLogin.jsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+  validate() {
+    let firstName = this.state.firstName
+    let lastName = this.state.lastName
+    let email = this.state.email
+    let password = this.state.password
+    let confirmPassword = this.state.confirmPassword
+    let valid = true
+
+
```
Comment (bwencke) [inline imports/ui/views/login/LoginModal/CreateAnAccountLogin.jsx]: why so many empty lines?

File: imports/ui/views/login/LoginModal/SocialLogin.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+
+import RaisedButton from 'material-ui/RaisedButton'
+import SvgIcon from 'material-ui/SvgIcon'
+
+export default class SocialLogin extends Component {
+
```
Comment (bwencke) [inline imports/ui/views/login/LoginModal/SocialLogin.js:7]: why so many empty lines?

File: imports/ui/views/login/LoginModal/index.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+export default class LoginModal extends Component {
+  render() {
+    return (
+      <Modal
+        isOpen={true}
+        contentLabel="Login/Register" 
+        style={loginModal} >
+          <div style={center}>
+            <span style={bigHeader}>Let's get to yo account</span>
```
Comment (bwencke) [inline imports/ui/views/login/LoginModal/index.js]: um...

File: imports/ui/views/login/LoginModal/index.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+            <div style={topLogin}>
+              <EmailPasswordLogin />
+              <div style={divider} />
+              <SocialLogin />
+            </div>
+            <div style={bigHeader}>Oh, you ain't got an account yet?</div>
+            <div style={smallHeader}>Waddup! Create yo account to start showin and sharin yo interests</div>
```
Comment (bwencke) [inline imports/ui/views/login/LoginModal/index.js]: Chelsie gave you the text to use here

Comment (garr741) [reply imports/ui/views/login/LoginModal/index.js]: Oh yeah, i forgot to change that back lol

**Review by bwencke — APPROVED** [review]

**Review by mhoc — APPROVED** [review]

**Pull request merged**